tests/integration-test.sh

```diff
@@ -47,7 +47,12 @@ rc="$?"
 timestamp=$(date +%s)
 outfile="integration_logs_"$DRONE_BUILD_NUMBER"_"$DRONE_COMMIT".zip"
 
-zip -9 $outfile output.xml log.html report.html package.list *container-logs.zip *.log
+if [ -f "/var/log/vic-machine-server/vic-machine-server.log" ]
+then
+  zip -9 $outfile output.xml log.html report.html package.list *container-logs.zip *.log /var/log/vic-machine-server/vic-machine-server.log 
+else
+  zip -9 $outfile output.xml log.html report.html package.list *container-logs.zip *.log
+fi
```

Comment:
zip actually just works on whatever file exists, so I don't think you need this new if/then block:
```
anon@anon-Latitude-E7450:~$ zip -9 test.zip test.robot doesntexit.log
	zip warning: name not matched: doesntexit.log
  adding: test.robot (deflated 59%)
anon@anon-Latitude-E7450:~$ ls test.zip
test.zip
```

Reply:
Awesome thanks!